frontend/src/metadata/MetadataPanel.svelte

```diff
@@ -306,6 +376,62 @@
 		<SliceCell slice={s} />
 	{/each}
 
+	<div id="tag-header" class="inline" style:margin-top="10px">
+		<div class="inline">
```

Comment:
I also can't decide how the sticky headers should work. If you scroll on the left you'll see the slice and metadata headers stick to the top. I think the tag one should too? But then it gets a bit busy... maybe we can ask erica